.github/actions/vtk-install-dep/action.yml

```diff
@@ -0,0 +1,121 @@
+name: 'Install VTK Dependency'
+description: 'Install VTK Dependency using cache when possible'
+inputs:
+  vtk_version:
+    description: 'VTK version'
+    required: true
+  vtk_sha_file:
+    description: 'VTK SHA file'
+    required: true
+  os:
+    description: 'OS to build on'
+    required: true
+runs:
+  using: "composite"
+  steps:
+
+    - name: Recover VTK Short SHA from file
+      if: |
+        steps.cache-vtk.outputs.cache-hit != 'true' &&
+        inputs.vtk_version == 'commit'
+      working-directory: ${{github.workspace}}/dependencies
+      shell: bash
+      run: echo "VTK_SHORT_SHA_OR_TAG=$(<${{ inputs.vtk_sha_file }})" >> $GITHUB_ENV
```

Comment:
If we store the long sha directly in the file instead, we could skipe the bare repo stuff enterelly.

wdyt @Meakk ?

Reply:
yup, store the long sha then

Reply:
not working for some reason ? dropping this.